src/Stratis.Bitcoin/Interfaces/IBlockDownloadState.cs

```diff
@@ -1,7 +1,23 @@
-﻿namespace Stratis.Bitcoin.Interfaces
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Stratis.Bitcoin.Interfaces
 {
     public interface IBlockDownloadState
```

Comment:
Add a bit of documentation here as well.

Reply:
Fixed.